infrastructure/ansible/roles/traffic_ops/defaults/main.yml

```diff
@@ -98,10 +98,13 @@ to_go_supported_ds_metrics:
   - tps_4xx
   - tps_5xx
 
+to_go_client_auth_enabled: 0
 to_go_whitelisted_oauth_urls: ""
 to_go_oauth_user_attribute: ""
 to_go_oauth_client_secret: ""
 
+to_client_cert_root_directory: ""
```

Comment:
How about a more sane default for `to_client_cert_root_directory` like `"/etc/pki/tls/traffic_ops/"`?

Reply:
done